drivers/gpio/Kconfig.max14917

```
@@ -0,0 +1,20 @@
+# Copyright (c) 2025 Analog Devices Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+# MAX14917 GPIO configuration options
+
+menuconfig GPIO_MAX14917
+	bool "MAX14917 GPIO driver"
+	default y
+	select CRC
+	depends on DT_HAS_ADI_MAX14917_GPIO_ENABLED && SPI
+	help
+	  Enabe MAX1417 octal industrial digital
```

Looks good to me. 
Just you may correct:
```suggestion
	  Enable MAX1417 octal industrial digital
```